manifests/java.yml

```diff
@@ -5,4 +5,10 @@ tests/:
         "*": "v0.87.0"
         akka-http: missing_feature (No AppSec support)
         spring-boot-3-native: missing_feature (GraalVM. Tracing support only)
-        spring-boot-payara: missing_feature (No AppSec support)
\ No newline at end of file
+        spring-boot-payara: missing_feature (No AppSec support)
+    iast/:
```

Comment:
`iast/` must be placed above `test_alpha.py`

I made a pretty agressive linter for this file , to help finding quickly the good location. Rules are : 

* folder must be placed before files
* then, name must be ordered alpabetically